src/js/game/systems/item_processor.js

```diff
@@ -23,7 +23,7 @@ export class ItemProcessorSystem extends GameSystemWithFilter {
             // First of all, process the current recipe
             processorComp.secondsUntilEject = Math_max(
                 0,
-                processorComp.secondsUntilEject - this.root.dynamicTickrate.deltaSeconds
+                processorComp.secondsUntilEject - this.root.dynamicTickrate.deltaSeconds,
```

Comment:
I think your IDE added a trailing comma there, however the prettier config has this explicitely disabled. Not a big issue, I can reformat the file after merging it.